samples/browser/src/main/java/org/mozilla/samples/browser/ExternalAppBrowserFragment.kt

```diff
@@ -68,7 +68,9 @@ class ExternalAppBrowserFragment : BaseBrowserFragment(), UserInteractionHandler
             requireActivity(),
             components.store,
             sessionId!!
-        )
+        ) {
+            // No-op. Client may override this
+        }
```

Comment:
Isn't that already the default behavior? Or do you just want to make this explicit here so that people reading the sample code will know about this?

Reply:
This piece of code is for a lambda parameter that I added and will be a breaking change for our clients who will now have to provide a specific behaviour for handling the error.